client_cli.go

```
@@ -0,0 +1,76 @@
+package client
+
+import (
+	"github.com/juju/errgo"
+	"github.com/coreos/fleet/job"
+
+	execPkg "os/exec"
+)
+
+const (
+	FLEETCTL        = "fleetctl"
+	ENDPOINT_OPTION = "--endpoint"
+	ENDPOINT_VALUE  = "http://172.17.42.1:4001"
+)
+
+type ClientCLI struct {
+	etcdPeer string
+}
+
+func NewClientCLI() FleetClient {
+	return NewClientCLIWithPeer(ENDPOINT_VALUE)
+}
+
+func NewClientCLIWithPeer(etcdPeer string) FleetClient {
+	return &ClientCLI{
+		etcdPeer: ENDPOINT_VALUE,
+	}
+}
+
+func (this *ClientCLI) Submit(name, filePath string) error {
+	cmd := execPkg.Command(FLEETCTL, ENDPOINT_OPTION, this.etcdPeer, "submit", filePath)
+	_, err := exec(cmd)
+
+	if err != nil {
+		return errgo.Mask(err)
+	}
+
+	return nil
+}
+
+func (this *ClientCLI) Get(name string) (*job.Job, error) {
+	return &job.Job{}, nil
```

Please return an error here.